vlog4j-client/src/test/java/org/vlog4j/client/PrintQueryResultsTest.java

```diff
@@ -0,0 +1,63 @@
+package org.vlog4j.client;
```

Comment:
why not have the same package name for test classes as for source classes?
Then, you can declare some methods package-protected and test them here, while making sure they cannot be accessed from elsewhere.

Reply:
Thank you. This was a silly mistake